src/screens/Home/index.tsx

```diff
@@ -1,71 +1,39 @@
-import React from 'react';
-import Button from '~/components/Button';
-import Input from '~/components/Input';
+import React, {useEffect} from 'react';
+import {TouchableOpacity} from 'react-native';
+import {useNavigation} from '@react-navigation/native';
+
+import {ArticleJSON} from '~/@types/dtos/everythingDTO';
+
+import Separator from '~/components/Separator';
 import Text from '~/components/Text';
 
 import {Home} from './styles';
+import useHomeController from './useHomeController';
 
 const HomeScreen = () => {
   const {Container} = Home;
+  const {navigate} = useNavigation();
+  const {fetchNews, news} = useHomeController();
 
-  return (
-    <Container>
-      <Button icon="share" isLoading={false} mode="contained" color="primary">
-        Entrar
-      </Button>
-
-      <Button isLoading mode="outlined" color="error">
-        Cadastre-se
-      </Button>
-
-      <Input icon="paperPlane" iconPosition="left" secureTextEntry />
+  useEffect(() => {
+    fetchNews();
+  }, [fetchNews]);
 
-      <Text typography="h1" accessibilityLabel="Ibravico">
-        H1
-      </Text>
-      <Text typography="h2" accessibilityLabel="Ibravico">
-        H2
-      </Text>
-      <Text typography="h3" accessibilityLabel="Ibravico">
-        H3
-      </Text>
-      <Text typography="h4" accessibilityLabel="Ibravico">
-        H4
-      </Text>
-      <Text typography="h5" accessibilityLabel="Ibravico">
-        H5
-      </Text>
-      <Text typography="h6" accessibilityLabel="Ibravico">
-        H6
-      </Text>
-      <Text typography="h7" accessibilityLabel="Ibravico">
-        H7
-      </Text>
-      <Text typography="h8" accessibilityLabel="Ibravico">
-        H8
-      </Text>
-      <Text typography="body1" accessibilityLabel="Ibravico">
-        BODY1
-      </Text>
-      <Text typography="body2" accessibilityLabel="Ibravico">
-        BODY2
-      </Text>
-      <Text typography="body3" accessibilityLabel="Ibravico">
-        BODY3
-      </Text>
-      <Text typography="caption" accessibilityLabel="Ibravico">
-        CAPTION
-      </Text>
-      <Text typography="overline" accessibilityLabel="Ibravico">
-        OVERLINE
-      </Text>
-      <Text typography="subtitle1" accessibilityLabel="Ibravico">
-        SUBTITLE1
-      </Text>
-      <Text typography="subtitle2" accessibilityLabel="Ibravico">
-        SUBTITLE2
-      </Text>
-    </Container>
+  return (
+    // TODO - Organize the homeScreen and fix lint problems
+    <Container
+      data={news}
+      ItemSeparatorComponent={() => <Separator height={18} />}
+      renderItem={({item}: {item: ArticleJSON}) => (
+        <TouchableOpacity
+          onPress={async () => {
+            navigate('InternalScreen', {item});
+          }}
+        >
+          <Text>{item.title}</Text>
```

Comment:
```suggestion
          <Text>{item?.title}</Text>
```

or create an if statement which returns nothing if `!item`